CRM/Contact/Import/Form/MapField.php

```diff
@@ -766,4 +740,35 @@ private function isSkipDuplicates(): bool {
     return ((int) $this->getSubmittedValue('onDuplicate')) === CRM_Import_Parser::DUPLICATE_SKIP;
   }
 
+  /**
+   * Get the fields to be highlighted in the UI.
+   *
+   * The highlighted fields are those used to match
+   * to an existing contact.
+   *
+   * @return array
+   *
+   * @throws \CRM_Core_Exception
+   */
+  private function getHighlightedFields(): array {
+    $entityFields = [
+      'Individual' => ['first_name', 'last_name'],
+      'Organization' => 'organization_name',
```

Comment:
```suggestion
      'Organization' => ['organization_name'],
      'Household' => ['household_name'],
```

Otherwise you get
`Error: [] operator not supported for strings in CRM_Contact_Import_Form_MapField->getHighlightedFields()` when it tries to append email to the array, which is not an array.

Reply:
opps! fixed